src/valgrind.supp

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This file doesn't get invoked by the unit test changes, but should be used when running the entire server under Valgrind for manual testing.

When I'm doing this, I typically use a command like:
`valgrind --leak-check=full --show-possibly-lost=no --suppressions=src/valgrind.supp src/redis-server [path/to/redis.conf]`

Reply:
Can we find a different location for that file, I wish to keep the src folder as clean as possible

Reply:
Where would you prefer? Redis and RediSearch both use their src folders, though the names are different in every project, so I don't think there's any real design standard.

I wouldn't mind having a root-level tools or utils folder (although we already have the latter under src) for this, mkdocs.yml, and perhaps the bulk insert script. Open to other options, though!

Reply:
NM, let's leave it there